models/staging/ga4/base/base_ga4__events_intraday.sql

```diff
@@ -20,7 +20,11 @@ with source as (
         event_bundle_sequence_id,
         event_server_timestamp_offset,
         user_id,
+<<<<<<< HEAD
+        user_pseudo_id as user_key,
+=======
```

Comment:
heads up here. I saw this in a previous PR as well that the merge conflict made it into the commit.

Reply:
fixed